### PR TITLE
noImplicitAny: Heatmap and TableRenderer

### PR DESCRIPTION
**Pull request by tskarhed**
#14714 

**Special notes for your reviewer**:
I added a types file for the heatmap data converter. Should it be in some other place? Also check the `Bucket` interface, maybe it can be used in other places as well?

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Just a merge conflict that needs to be resolved, otherwise looks good!

File: package.json [hunk 1]
```diff
@@ -189,6 +189,7 @@
   "dependencies": {
     "@babel/polyfill": "7.2.5",
     "@torkelo/react-select": "2.4.1",
+    "@types/angular-route": "1.7.0",
```
Comment (dprokop) [inline package.json]: 👍

File: packages/grafana-ui/src/types/datasource.ts [hunk 2]
```diff
@@ -370,7 +370,7 @@ export interface DataQueryError {
 }
 
 export interface ScopedVar {
-  text: any;
+  text?: any;
```
Comment (dprokop) [inline packages/grafana-ui/src/types/datasource.ts]: @torkelo is this change safe?

Comment (dprokop) [reply packages/grafana-ui/src/types/datasource.ts]: ight, I feel like introducing optional here is not neccessary. Main driver here is table/renderer.ts / renderRowVariables. I suggest using stringified value there as `text` property value

**Pull request merged**